pkgs/development/tools/godot/mono-deps.nix

```diff
@@ -0,0 +1,152 @@
+{ fetchurl }: let
```

Comment:
I am not that familiar with mono in nix but you used a tool to generate this? If so we should probably add an update script for it.

Reply:
I have added a `generate-mono-deps.sh` script to generate this file.